core/src/services/email.rs

```diff
@@ -35,6 +37,11 @@ impl From<SendGrid> for Email {
 impl EmailNotification for Email {
     #[tracing::instrument]
     async fn send(&self, email: String, message: EmailMessage) -> Result<(), SendEmailError> {
+        if !self.enabled {
+            tracing::warn!("Email service is disabled!");
```

Comment:
Maybe it is better to print message content to stdout?

Reply:
Alongside warning with a level of DEBUG - yeah, why not.